packages/frontend/core/src/hooks/affine/use-share-url.ts

```diff
@@ -21,9 +22,8 @@ const useGenerateUrl = ({ workspaceId, pageId, urlType }: UseSharingUrl) => {
   // to generate a public url like https://app.affine.app/share/123/456
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 18]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This hook is used for the sharing panel and it seems feature-wise we do not need to sync the blockid into hash there.

Some implementation issues:
Consdering hash is the same with blockId, which should also be part of the `UseSharingUrl` props. Also, the given active editor may not necessarily the same with the given pageId.
Thus the hook may be refactored to a plain function.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7339?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
I made some changes according to your suggestions. What do you think about this?